docs/src/components/components/action/button-outline/README.md

```diff
@@ -0,0 +1,23 @@
+
+# Button Outline
+
+The **button outline** component is used when you want the user to take an
+action. Buttons should be used for changing the state of a page or transforming
+data.
+
+## Use
+
+To use the **button outline** component, add the classes
+`usa-button usa-button-outline action` to a
+`<button>` element.
+
+In addition, add any of the following classes to the button to style the button
+based on status.
+
+| Class   | Description |
+| ------   | -----------|
+| `action-primary` | When to use |
+| `action-base`       | When to use |
+| `action-warning`  | When to use |
+| `action-cautious`    | When to use |
+| `action-finish`    | When to use |
```

Comment:
`warning`, `primary`, `finish`, and `base` are the allowed values in the Action `style` prop. I think `cautious` can be nixed.